lib/core/options.js

```diff
@@ -97,16 +97,9 @@ function resolveTracingOptions (options, config) {
   /** @type {NonNullable<import('../../types/sentry').TracingConfiguration>} */
   const tracingOptions = merge(
     {
-      tracesSampleRate: 1.0,
-      vueOptions: {
-        tracing: true,
-        tracingOptions: {
-          hooks: ['mount', 'update'],
-          timeout: 2000,
-          trackComponents: true,
-        },
-      },
-      browserOptions: {},
+      tracesSampleRate: 1,
+      trackComponents: true,
+      browserTracing: {},
```

Comment:
I'm thinking now that the `vueOptions` should still be separated since the tracing options are also used on the server-side while options like `trackComponents` and `hooks` are Vue-only.

Reply:
Addressed in 19357615abab42136c488e459bb4ba62eded8217